app/src/main/java/explorer/city/com/cityexplorer/view/activity/MainActivity.kt

```diff
@@ -0,0 +1,71 @@
+package explorer.city.com.cityexplorer.view.activity
+
+import android.arch.lifecycle.LifecycleActivity
+import android.arch.lifecycle.Observer
+import android.arch.lifecycle.ViewModelProviders
+import android.os.Bundle
+import android.support.v7.widget.LinearLayoutManager
+import android.support.v7.widget.RecyclerView
+import android.util.Log
+import com.jakewharton.rxbinding2.widget.RxTextView
+import explorer.city.com.cityexplorer.R
+import explorer.city.com.cityexplorer.model.SearchItem
+import explorer.city.com.cityexplorer.view.adapter.SearchAdapter
+import explorer.city.com.cityexplorer.viewModel.MainViewModel
+import io.reactivex.disposables.CompositeDisposable
+import kotlinx.android.synthetic.main.activity_main.*
+import java.util.concurrent.TimeUnit
+
+class MainActivity : LifecycleActivity() {
+
+    companion object {
+        val TAG: String = "MainActivity"
```

Comment:
I guess its because you want it to be `static` -- in which case you could remove the companion object and replace with:

`private static val TAG = "MainActivity"`